src/dispatching.rs

```diff
@@ -187,12 +187,36 @@
 //! useful features. See [`examples/dispatching_features.rs`] as a more involved
 //! example.
 //!
+//! ## Dispatching or REPLs?
+//!
+//! The difference between dispatching and the REPLs ([`crate::repl`] & co) is
+//! that dispatching allows you a greater degree of flexibility at the cost of a
```

Comment:
"allows you a greater degree"?  I think you can allow an action "allows you to do X", but not a degree. Maybe
```suggestion
//! that dispatching gives you a greater degree of flexibility at the cost of a
```